analyzer-models/src/main/java/ffb/analyzer/models/espn/TeamScore.java

```diff
@@ -12,12 +12,14 @@
     private int gamesPlayed;
     private int teamId;
 
-    @JsonProperty("totalPoints")
-    private float points;
-
     @JsonProperty("pointsByScoringPeriod")
     private Map<Integer, Float> scores;
 
+    public float getTotalPointsScored() {
```

Comment:
Should we cache this calculation?

Reply:
Yes, given the volume of scores we could be aggregating I think it makes sense to cache this off.